Carthage fails instantly on install

To install the library[ in my sample project](https://github.com/ShadeApps/Atals-Cartahge-Test), I do as it's instructed in the manual to install it via Carthage. 
[The output log](https://www.dropbox.com/s/ctwwijb5pbk5l0f/carthage-xcodebuild.NFxzXr.log?dl=1) which I'm getting basically says:

```
/Users/grischyov/Desktop/Atals Cartahge Test/Carthage/Checkouts/Atlas-iOS/Code/Controllers/ATLConversationListViewController.h:22:9: fatal error: module 'LayerKit' not found
@import LayerKit;
 ~~~~~~~^~~~~~~~
1 error generated.

** BUILD FAILED **


The following build commands failed:
	CompileC /Users/grischyov/Library/Caches/org.carthage.CarthageKit/DerivedData/Atlas-iOS/v1.0.32/Build/Intermediates/Atlas.build/Release-iphoneos/Atlas.build/Objects-normal/armv7/Atlas.o Code/Atlas.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

Other libraries in other projects using Carthage compile and build just fine, so I think it might as well by Layer's issue.

I'm using Xcode 8.3.2 (8E2002) and latest stable version of Carthage.

--- a/Code/Controllers/ATLConversationListViewController.h
+++ b/Code/Controllers/ATLConversationListViewController.h
@@ -19,7 +19,7 @@
 //
 
 #import <UIKit/UIKit.h>
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 #import "ATLConversationTableViewCell.h"
 #import "ATLAvatarItem.h"
 #import "ATLParticipant.h"

--- a/Code/Controllers/ATLConversationViewController.h
+++ b/Code/Controllers/ATLConversationViewController.h
@@ -19,7 +19,7 @@
 //
 
 #import <UIKit/UIKit.h>
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 #import <MapKit/MapKit.h>
 #import "ATLParticipant.h"
 #import "ATLBaseConversationViewController.h"

--- a/Code/Models/ATLConversationDataSource.h
+++ b/Code/Models/ATLConversationDataSource.h
@@ -19,7 +19,7 @@
 //
 
 #import <UIKit/UIKit.h> 
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 
 extern NSInteger const ATLNumberOfSectionsBeforeFirstMessageSection;
 

--- a/Code/Models/ATLDataSourceChange.h
+++ b/Code/Models/ATLDataSourceChange.h
@@ -19,7 +19,7 @@
 //
 
 #import <Foundation/Foundation.h>
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 
 NS_ASSUME_NONNULL_BEGIN
 @interface ATLDataSourceChange : NSObject

--- a/Code/Protocols/ATLConversationPresenting.h
+++ b/Code/Protocols/ATLConversationPresenting.h
@@ -19,7 +19,7 @@
 //
 
 #import <Foundation/Foundation.h>
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 #import "ATLAvatarItem.h"
 
 NS_ASSUME_NONNULL_BEGIN

--- a/Code/Protocols/ATLMessagePresenting.h
+++ b/Code/Protocols/ATLMessagePresenting.h
@@ -20,7 +20,7 @@
 
 #import <Foundation/Foundation.h>
 #import "ATLParticipant.h"
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 
 NS_ASSUME_NONNULL_BEGIN
 /**

--- a/Code/Utilities/ATLMessagingUtilities.h
+++ b/Code/Utilities/ATLMessagingUtilities.h
@@ -19,7 +19,7 @@
 //
 
 #import <Foundation/Foundation.h>
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 #import <MapKit/MapKit.h>
 #import <ImageIO/ImageIO.h>
 #import "ATLMediaAttachment.h"

--- a/Code/Views/ATLBaseCollectionViewCell.h
+++ b/Code/Views/ATLBaseCollectionViewCell.h
@@ -11,7 +11,7 @@
 #import "ATLAvatarImageView.h"
 #import "ATLMessagePresenting.h"
 
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 
 typedef NS_ENUM(NSInteger, ATLCellType) {
     ATLOutgoingCellType,

--- a/Code/Views/ATLMessageCollectionViewCell.m
+++ b/Code/Views/ATLMessageCollectionViewCell.m
@@ -24,7 +24,7 @@
 #import "ATLIncomingMessageCollectionViewCell.h"
 #import "ATLOutgoingMessageCollectionViewCell.h"
 
-@import LayerKit;
+#import <LayerKit/LayerKit.h>
 
 
 NSString *const ATLGIFAccessibilityLabel = @"Message: GIF";

--- a/Examples/Programmatic/ProgrammaticAppDelegate.m
+++ b/Examples/Programmatic/ProgrammaticAppDelegate.m
@@ -10,7 +10,7 @@
 #import "ATLSampleConversationListViewController.h"
 #import "LayerKitMock.h"
 #import <Atlas/Atlas.h>
-@import LayerKit; 
+#import <LayerKit/LayerKit.h> 
 
 static BOOL ATLIsRunningTests()
 {

